src/Bicep.Core/Parser/StringUtils.cs

```diff
@@ -42,5 +45,7 @@ public static string EscapeBicepString(string value)
 
             return buffer.ToString();
         }
+
+        public static bool IsMultilineString(string value) => NewLineRegex.Matches(value).Count >= 2;
```

Comment:
>IsMultilineString [](start = 27, length = 17)

So something like "\nstuff" or "stuff\n" is considered a single line string?

Reply:
Hmm...good question. I was bad at naming stuff. Will change it to something like `HasAtLeastTwoNewlines`.

Reply:
No worries - new name works.